doc/development/Makefile

```diff
@@ -0,0 +1,9 @@
+MODULE_TOPDIR = ..
+
+include $(MODULE_TOPDIR)/include/Make/Other.make
+
+# So far, we disntinguished user and contributor documentation here by
+# extension. This is no longer possible with Markdown.
+MDFILES := github_guide.md style_guide.md
```

Comment:
Let's move this whole Makefile and the task of publishing development-related Markdown files to another PR or issue.